nvidia-nano/GiveMeWES.yaml

```diff
@@ -0,0 +1,597 @@
+---
+- name: Check Jetpack version
+  hosts: all
+  vars_prompt:
+
+    - name: Connect Drive
+      prompt: Please connect your usb drive to the nano. When ready press enter
+      private: no
+
+  tasks:
+    - name: Update repositories cache
+      ansible.builtin.apt:
+        update_cache: yes
+
+    - name: Register jetpack version
+      command: apt-cache show nvidia-jetpack
+      register: jetpackVer
+
+    - name: Test for Jetpack 4.4.1
+      fail: msg="The wrong version of Jetpack was flashed. I want jetpack 4.4.1"
+      when: not ('4.4.1' in jetpackVer.stdout)
+
+- name: Set Up Extra Drive
+  hosts: all
+  tasks:
+    - name: Clear the current partition table and create GPT table
+      shell: echo -e "g\nw" | fdisk --wipe always --wipe-partitions always /dev/sda #Can't use parted module because it has issues clearing partitions in use
+      args:
+        executable: /bin/bash
+
+    - name: Make the SWAP partition (16GB SWAP) #Can't create swap partition with parted module so use shell command
+      shell: echo -e "n\n1\n2048\n+16G\nt\n19\nw" | fdisk --wipe always --wipe-partitions always /dev/sda
```

Comment:
look into @seanshahkarami comment in Slack about how the 19 (for the label) may not match between os versions.  probably doesnt matter as we know jetpack 4.4.1 is consistent